spotbugs-tests/build.gradle

```diff
@@ -14,21 +18,52 @@ dependencies {
   implementation 'junit:junit:4.13.1'
   implementation 'org.hamcrest:hamcrest-all:1.3'
   implementation 'org.apache.ant:ant:1.10.12'
+  implementation "org.apache.logging.log4j:log4j-slf4j18-impl:$log4jVersion"
+  implementation 'com.google.errorprone:error_prone_annotations:2.10.0'
+  implementation files(project(":spotbugs").sourceSets.gui.output)
 }
 
 java {
   withSourcesJar()
 }
 
+
+tasks.named('compileJava', JavaCompile).configure {
+  options.release = 11
+}
+
 artifacts {
   archives sourcesJar
 }
 
 tasks.named('jacocoTestReport', JacocoReport).configure {
+  dependsOn('unstableTest')
   additionalSourceDirs.setFrom files(project(':spotbugs').sourceSets.main.java.srcDirs)
   additionalClassDirs.setFrom files(project(':spotbugs').sourceSets.main.output.classesDirs)
+  executionData.setFrom files("$buildDir/jacoco/unstableTest.exec", "$buildDir/jacoco/test.exec")
 }
 
+// Tests below fail if executed with other tests
```

Comment:
It's strange, could you share how they fail?

Reply:
see https://github.com/spotbugs/spotbugs/issues/1906#issuecomment-1007739204